packages/shared/src/components/Feed.tsx

```diff
@@ -371,6 +371,8 @@ export default function Feed<T>({
       );
     },
     post,
+    prevPost: (items[postMenuIndex - 1] as PostItem)?.post,
+    nextPost: (items[postMenuIndex + 1] as PostItem)?.post,
```

Comment:
Oef not against the implementation, but thinking about a leanFeed, this is yet another constraint in moving it.